src/jira/client/jira-client.ts

```diff
@@ -348,6 +367,20 @@ const withinIssueKeyLimit = (resources: { issueKeys: string[] }[]): boolean => {
 	return Math.max(...issueKeyCounts) <= ISSUE_KEY_API_LIMIT;
 };
 
+/**
+ * Returns if the max length of the issue key field is within the limit
+ * Assumption is that the transformed resource only has one association which is for
+ * "issueIdOrKeys" association.
+ */
+const withinIssueKeyAssociationsLimit = (resources: JiraRemoteLink[]): boolean => {
+	if (!resources) {
```

Comment:
if it's possible for resources to be undefined, the type needs to be `resources?:JiraRemoteLink[]`

Reply:
Same as above.